dali/test/python/test_operator_warp.py

```diff
@@ -219,3 +220,45 @@ def test_gpu_vs_cpu():
         gpu_pipeline.build();
 
         compare(cpu_pipeline, gpu_pipeline, 1)
+
+def _test_extremely_large_data(device):
+  in_size = 30000
+  out_size = 10
+  channels = 3
+
+  def get_data():
+    out = np.full([in_size, in_size, channels], 42, dtype=np.uint8)
+    for c in range(channels):
+      out[in_size-1, in_size-1, c] = c
+    return [out]
+
+  pipe = Pipeline(1, 3, 0)
```

Comment:
I would set prefetch_queue_depth to 1 to reduce the number of output buffers.

Reply:
Done.